notebooks/time-to-merge-prediction/thoth-station/thoth_station_feature_engg.ipynb

```diff
@@ -0,0 +1,5596 @@
+{
```

Comment:
<p>Hmm so looking at the <a href=" https://github.com/thoth-station/graph-refresh-job/pull/672" target="_blank">PR</a> in this case, it doesn't seem like sesheta interacted with it 257 times. I could be misremembering but I believe interactions stores the number of words commented on a PR by a user, and not necessarily the number of times they interacted.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/496/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>sounds good. ill update this. </p>